libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -146,6 +146,7 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @DisplayName: Arming options
     // @Description: Options that can be applied to change arming behaviour
     // @Bitmask: 0:Disable prearm display,1:Do not send status text on state change
+    // @Bitmask{Copter}: 0:Disable prearm display,1:Do not send status text on state change,2:Require position for arming
```

Comment:
```suggestion
    // @Bitmask{Copter}: 0:Disable prearm display,1:Do not send status text on state change,2:Require location for arming
```

Reply:
Why not add it to plane also?

Reply:
Plane *already* requires position for arming, all the time.

That's actually why we put this into Copter's flight options in the first place - we're going to have an awkward time when Plane no longer requires position for arming....

But we want this bit for Rover, so @rmackay9 asked for it to be moved (https://github.com/ArduPilot/ardupilot/pull/29073)